### ISSUE TITLE
Intermittent test error: Unhandled error. (TypeError: require(...) is not a function

### ISSUE THREAD

**Issue by jacoscaz** [comunica/comunica#792]
#### Issue type:

- :bug: Bug

____
#### Description:

While running `yarn run test` or `yarn run test-ci`, I sometimes get one or two failures unrelated to what I am working on. These always disappear as soon as I launch the test suite again and, although they may be thrown by different tests, share the following characteristics:

- the error thrown is a `TypeError` related to the invocation of something that is not a function: `Unhandled error. (TypeError: require(...) is not a function`;
- the error thrown always references a `.toNodeReadable` function in the first few lines of its stack.

The following is an example of one such failure:

```
 FAIL  packages/actor-rdf-resolve-quad-pattern-hypermedia/test/LinkedRdfSourcesAsyncRdfIterator-test.ts
  ● LinkedRdfSourcesAsyncRdfIterator › A LinkedRdfSourcesAsyncRdfIterator instance › handles metadata event emitted before the end event

    Unhandled error. (TypeError: require(...) is not a function

      at Function.toNodeReadable (packages/bus-http/lib/ActorHttp.ts:376:32)
      at ActorRdfDereferenceHttpParse.run (packages/actor-rdf-dereference-http-parse/lib/ActorRdfDereferenceHttpParseBase.ts:1480:53)
      at MediatedLinkedRdfSourcesAsyncRdfIterator.getSource (packages/actor-rdf-resolve-quad-pattern-hypermedia/lib/MediatedLinkedRdfSourcesAsyncRdfIterator.ts:905:32))Error [ERR_UNHANDLED_ERROR]: Unhandled error. (TypeError: require(...) is not a function
      at Function.toNodeReadable (packages/bus-http/lib/ActorHttp.ts:376:32)
      at ActorRdfDereferenceHttpParse.run (packages/actor-rdf-dereference-http-parse/lib/ActorRdfDereferenceHttpParseBase.ts:1480:53)
      at MediatedLinkedRdfSourcesAsyncRdfIterator.getSource (packages/actor-rdf-resolve-quad-pattern-hypermedia/lib/MediatedLinkedRdfSourcesAsyncRdfIterator.ts:905:32))
      at SimpleTransformIterator.<anonymous> (packages/actor-rdf-resolve-quad-pattern-federated/lib/FederatedQuadSource.ts:3039:19)
      at MediatedLinkedRdfSourcesAsyncRdfIterator.destinationEmitError (node_modules/asynciterator/dist/asynciterator.cjs:1067:23)
      at node_modules/asynciterator/dist/asynciterator.cjs:177:26
      at MediatedLinkedRdfSourcesAsyncRdfIterator._destroy (node_modules/asynciterator/dist/asynciterator.cjs:868:9)
      at MediatedLinkedRdfSourcesAsyncRdfIterator.destroy (node_modules/asynciterator/dist/asynciterator.cjs:174:18)
      at packages/actor-rdf-resolve-quad-pattern-hypermedia/lib/LinkedRdfSourcesAsyncRdfIterator.ts:200:30
```

I have not (yet?) found a way to deterministically reproduce these failures.

____
#### Environment:

| software            | version
| ------------------- | -------
| Comunica Init Actor | 1.19.2 (dev)
| node                | v14.15.4
| npm                 | 7.4.1
| yarn                | 1.22.10
| Operating System    | darwin (Darwin 20.3.0)

**Comment by rubensworks** [comment]
I just noticed that I also get this error, but it does not seem to fail any tests on my end though.

In any case, something to must be fixed.

**Comment by rubensworks** [comment]
Fixed in f7ce061aab669a261a4a0dbbc8c88956739504a1

**Comment by jacoscaz** [comment]
Just tried running the test suite a few times with no errors - nice!
